src/sql/parts/notebook/outputs/renderers.ts

```diff
@@ -0,0 +1,630 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+import { default as AnsiUp } from 'ansi_up';
```

Comment:
Minor: I think you can change to `import AnsiUp from 'ansi_up';`?

Reply:
We can't change it as it was exported as default class in ansi_up module.